iot-edge/EdgeSolution/modules/CSharpModule/Program.cs

```diff
@@ -0,0 +1,172 @@
+namespace CSharpModule
+{
+    using System;
```

Comment:
Please use the SDK code standards. The `using` statements should be outside of the namespace.